README.md

```diff
@@ -116,6 +116,22 @@ docker pull ghcr.io/hadolint/hadolint:latest-debian
 docker pull ghcr.io/hadolint/hadolint:latest-alpine
 ```
 
+You also can install it in Dockerfile
+
+```dockerfile
+# Pull one of the images above
+FROM ghcr.io/hadolint/hadolint as hadolint
```

Comment:
```suggestion
FROM ghcr.io/hadolint/hadolint AS hadolint
```
The `AS` term is a Dockerfile keyword.
* https://docs.docker.com/build/building/multi-stage/#name-your-build-stages
* https://docs.docker.com/reference/dockerfile/#from